src/main/webapp/index.html

```diff
@@ -10,8 +10,25 @@ <h1 class="name">Events</h1>
     <div class="list">
     </div>
 
-    <div class="add">
+    <div id="addButton">
         <button>+</button>
     </div>
+
+    <div id="popup-window">
+        <form>
+            <p>Add Event</p>
+
+            <label>Title:</label>
+            <input type="text" placeholder="Enter event title" required>
+
+            <label>Location:</label>
+            <input type="text" placeholder="Enter event location" required>
+
+            <label>Starting Time:</label>
```

Comment:
Start time?

Reply:
For the event Breakfast we will have: starting time: 9AM, and ending time 10AM

Reply:
I assume this will be easier to manage and also easier to insert the time in google maps